### DISCUSSION TITLE
loose Wlan connection frequently ...  Software Issue !

### DISCUSSION THREAD

**Discussion by sat-fan2019** [jomjol/AI-on-the-edge-device#1372] [Q&A]
I loose the network connection frequently.
I  changed wlan extender, router ect... dns or fixed IP
I changed as well the SD card and flash the esp again.
Now I checked it 1 m distance to the router with a ping.
I checked it with a second cam32 Module. Same Problems.
Next I use the hardware for a direct esphome integration. that takes a pic. each 8 seconds. No Problems there !
So HW should be fine !
Any Idea ?
 Thx
![Eingabeaufforderung 17 11 2022 22_26_44](https://user-images.githubusercontent.com/48486423/202563875-9ffe20bb-ba0d-4f5b-b351-1c4834036061.png)

**Comment by maurcabezas** [comment]
I post the same problem #1446 , it was working very well for months, but now in my case after updating, I started with the same problem.. I hope somebody can help.

**Reply by caco3** [reply]
Which version arre you using?

Do you use MQTT?
There is a stability issue which can cause the device to start hanging.
Restart it and check the Log!

You could try the latest rolling which hopefully brings improvement, see https://github.com/jomjol/AI-on-the-edge-device/issues/1455
